### PR TITLE
[Hotfix] Hotfix for issue #122

### PR DESCRIPTION
**Pull request by DavdGao**
As the title says

### CODE REVIEW

**Comment by yxcdy** [comment]
LGTM

**Comment by yxdyc** [comment]
LGTM.

File: federatedscope/core/trainers/trainer_nbafl.py [hunk 1]
```diff
@@ -54,8 +54,10 @@ def init_nbafl_ctx(base_trainer):
     cfg = base_trainer.cfg
 
     # set proximal regularizer
+    cfg.regularizer.defrost()
     cfg.regularizer.type = 'proximal_regularizer'
     cfg.regularizer.mu = cfg.nbafl.mu
+    cfg.regularizer.freeze(save=False)
```
Comment (joneswong) [inline federatedscope/core/trainers/trainer_nbafl.py:60]: when should we set `save` to be `False`?

**Review by joneswong — DISMISSED** [review]
looks good. plz resolve the conflicts. just a kind reminder: whether FedEx client needs to set `save` to be `False`

File: federatedscope/core/trainers/trainer_nbafl.py (hunk 1 above)
Comment (DavdGao) [reply federatedscope/core/trainers/trainer_nbafl.py:60]: In my opinion, the target of saving configuration is to reproduct the experiment. So maybe we should save the `config`  before calling `FedRunner().run()`

**Comment by DavdGao** [comment]
> looks good. plz resolve the conflicts. just a kind reminder: whether FedEx client needs to set `save` to be `False`

Thanks, the conflicts are resolved.

File: federatedscope/core/trainers/trainer_nbafl.py (hunk 1 above)
Comment (xieyxclack) [reply federatedscope/core/trainers/trainer_nbafl.py:60]: But the config might be changed to "valid" value in some assertions and saved. I think the saved config implies what takes effect during the exp.

Comment (DavdGao) [reply federatedscope/core/trainers/trainer_nbafl.py]: > 

I have checked the code, and delete the `save` parameter in `cfg.freeze()`.

**Review by xieyxclack — APPROVED** [review]
It looks good to me. Thanks a lot @DavdGao

**Pull request merged**